Marlin/Configuration_adv.h

```diff
@@ -762,14 +762,14 @@
 
   /**
    * Screw thread:
-   *   M3: 0 = Clockwise, 1 = Counter-Clockwise
-   *   M4: 2 = Clockwise, 3 = Counter-Clockwise
-   *   M5: 4 = Clockwise, 5 = Counter-Clockwise
+   *   M3: 30 = Clockwise, 31 = Counter-Clockwise
+   *   M4: 40 = Clockwise, 41 = Counter-Clockwise
+   *   M5: 50 = Clockwise, 51 = Counter-Clockwise
    *
    * Default is Clockwise M3 (0). Most printers use M3
    * screws, with clockwise turns moving the bed down.
```

Comment:
Does `bed down` mean away from the nozzle? Thinking about printers where the bed moves up/down vs the nozzle moving up/down. Am I overthinking it?

Reply:
Only if you'd consider the nozzle hanging tip top below the bed, `bed down` would decrease the distance of bed and nozzle. If the bed or the nozzle is the motorized part does not matter.